exporter/sumologicexporter/factory.go

```diff
@@ -0,0 +1,60 @@
+// Copyright 2020 OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package sumologicexporter
+
+import (
+	"go.opentelemetry.io/collector/component"
+	"go.opentelemetry.io/collector/config/configmodels"
+	"go.opentelemetry.io/collector/exporter/exporterhelper"
+)
+
+const (
+	// The value of "type" key in configuration.
+	typeStr = "sumologic"
+)
+
+// NewFactory returns a new factory for the sumologic processor.
+func NewFactory() component.ExporterFactory {
+	return exporterhelper.NewFactory(
```

Comment:
Better use the helper methods like here: https://github.com/open-telemetry/opentelemetry-collector/blob/master/exporter/otlpexporter/factory.go#L33-L38

Reply:
I think I use it as in example, the rest of the methods are going to be part of another PR (exporter implementation)
https://github.com/SumoLogic/opentelemetry-collector-contrib/compare/drosiek-sumologic-output-init-pr...SumoLogic:drosiek-sumologic-output-queues-pr?expand=1